ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -20132,9 +20133,9 @@ periodics:
       - --oauth-token-path=/usr/local/github-credentials/oauth
       - --report-credentials-file=/etc/report/credentials
       - --secret-dir=/secrets/ci-pull-credentials
-      - --secret-dir=/usr/local/aws-ipi-p1-f14-cluster-profile
-      - --target=aws-ipi-p1-f14
-      - --variant=amd64-nightly-4.14-upgrade-from-stable-4.11
+      - --secret-dir=/usr/local/nutanix-ipi-proxy-fips-p2-f28-cluster-profile
+      - --target=nutanix-ipi-proxy-fips-p2-f28
```

Comment:
There are known [issue](https://issues.redhat.com/browse/OCPQE-15938) for 4.12-4.13 upgrade on Nutanix.  Maybe we could delay the eus test on such platform.

Reply:
Good suggestion! Lower the test frequency to f360 and let's move it back after nutanix upgrade step is ready.